### PR TITLE
Somfy open api

### PR DESCRIPTION
**Pull request by tetienne**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#19548

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Please merge the `cover.somfy.markdown` inside the `somfy.markdown`

**Review by frenck — CHANGES_REQUESTED** [review]
Hi @tetienne, could you please resolve the request by @klaasnicolaas? That allows us to merge this PR in and finish up the docs for 0.95

**Comment by tetienne** [comment]
Sorry I didn't see your message @klaasnicolaas @frenck. Cover documentation has been merged.

**Comment by frenck** [comment]
Thanks, @tetienne for that extreme quick response there! 🐎 

I'll take a look in a second.

**Comment by tetienne** [comment]
You are welcome 😊

File: source/_components/somfy.markdown [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: somfy.png
+ha_category:
+  - Hub
+ha_iot_class: Cloud Polling
+ha_release: 0.95
+ha_qa_scale: gold
```
Comment (frenck) [inline source/_components/somfy.markdown]: The quality scale index is incorrect IMHO. For a gold standard, there should be tests between HA & the integration. Currently, I only see tests for the configuration flow. (which are excluded from the code coverage, which it should NOT have been btw...)

See: https://developers.home-assistant.io/docs/en/integration_quality_scale_index.html

For now, I suggest removing this.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_components/somfy.markdown (hunk 1 above)
Comment (tetienne) [reply source/_components/somfy.markdown]: Are you talking about this: 
>Tests for reading data from/controlling the integration (docs)

What do you think to replace by Silver :2nd_place_medal: ?

Comment (tetienne) [reply source/_components/somfy.markdown]: For the coverage, I will see how include my tests.

Comment (frenck) [reply source/_components/somfy.markdown]: Well for silver it should:

✅  Connection/configuration is handled via a component.
❌  Set an appropriate SCAN_INTERVAL (if a polling integration)
❌ Raise PlatformNotReady if unable to connect during platform setup (if appropriate)
❓ Handles expiration of auth credentials. Refresh if possible or print correct error and fail setup.
❓ If based on a config entry, should trigger a new config entry flow to re-authorize.
❓ Handles internet unavailable. Log a warning once when unavailable, log once when reconnected.
❓ Handles device/service unavailable. Log a warning once when unavailable, log once when reconnected.
❌ Set available property to False if appropriate (docs)
✅ Entities have unique ID (if available) (docs)

Even though I haven't checked on the question marks,  I still see missing things for a silver 🥈

Comment (tetienne) [reply source/_components/somfy.markdown]: You are right. I will remove it. I have still a lot of stuff to do it seems. 
Sadly, it seems that existing component with qa scale don't respect these rules. Do you know a good one?

Comment (frenck) [reply source/_components/somfy.markdown]: The QA scale should be correct in the documentation. If you feel like one is incorrect, feel free to raise an issue or open a PR.

Hue & deCONZ are probably good examples.

**Review by frenck — APPROVED** [review]
This looks ready to merge to me!
Thanks, @tetienne 🎉

File: source/_components/somfy.markdown (hunk 1 above)
Comment (tetienne) [reply source/_components/somfy.markdown]: Thank you for the information.

**Pull request merged**